eng/common/TestResources/README.md

```diff
@@ -26,17 +26,10 @@ Note that `-Subscription` is an optional parameter but recommended if your accou
 is a member of multiple subscriptions.
 
 ```powershell
-Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
-$sp = New-AzADServicePrincipal -Role Owner
-eng\common\TestResources\New-TestResources.ps1 `
-  -BaseName 'myusername' `
-  -ServiceDirectory 'search' `
-  -TestApplicationId $sp.ApplicationId `
-  -TestApplicationSecret (ConvertFrom-SecureString $sp.Secret -AsPlainText)
+eng\common\TestResources\New-TestResources.ps1 -ServiceDirectory 'search'
```

Comment:
Since partner teams will not have access to Playground subscription, should we leave the first line with Connect-AzAccount?

Reply:
Or should we prompt them to enter a subscription after the fact?

Reply:
What do you think is more user friendly?

Reply:
I think I'll add the first line back

Reply:
If there was a way that we can try to login and only prompt if it failed, I would vote for that.